CONTRIBUTING.md

```diff
@@ -1,23 +1,81 @@
-# Holes
+# Creating holes
+
+## Open a GitHub Issue
+
+Create an issue and describe the hole
+
+## Add hole description to `holes.toml`
+
 Each hole needs to have its section in the `/config/holes.toml` file.
 Each section needs to define the following fields:
-* `category` - One of 'Art', 'Computing', 'Gaming', 'Mathematics', 'Sequence', 'Transform'.
-* `preamble` - HTML description of the hole. May include go templating marks. The root `.` is the data from the `data` field.
+
+- `category` - One of 'Art', 'Computing', 'Gaming', 'Mathematics', 'Sequence', 'Transform'.
+- `preamble` - HTML description of the hole. May include go templating marks. The root `.` is the data from the `data` field.
 
 In addition, each section may define the following fields:
-* `data` - JSON describing the data to be copied upon pressing "Copy as JSON". Relevant mainly for holes in the "Transform" category.
-* `links` - List of tables with keys `name` and `url`.
-* `experiment` - ID of the issue that suggested this hole or `-1` if the issue doesn't exist. Only defined for experimental holes.
-* `variants` - List of names of the holes that are variants of this hole, including itself.
+
+- `data` - JSON describing the data to be copied upon pressing "Copy as JSON". Relevant mainly for holes in the "Transform" category.
+- `links` - List of tables with keys `name` and `url`.
+- `experiment` - ID of the issue that suggested this hole or `-1` if the issue doesn't exist. Only defined for experimental holes.
+- `variants` - List of names of the holes that are variants of this hole, including itself.
+
+Example:
+
+```
```

Comment:
Can we add `toml` to the fenced code block opening tag? I suspect GitHub will give us lovely syntax highlighting then.